v2/api/batch/versions_matrix.md

```diff
@@ -1,35 +1,23 @@
-| EncryptionProperties_Status                                | v1alpha1api20210101 |
-| KeyVaultProperties                                         | v1alpha1api20210101 |
-| KeyVaultProperties_Status                                  | v1alpha1api20210101 |
-| KeyVaultReference                                          | v1alpha1api20210101 |
-| KeyVaultReference_Status                                   | v1alpha1api20210101 |
-| PoolAllocationMode_Status                                  | v1alpha1api20210101 |
-| PrivateEndpointConnectionPropertiesStatusProvisioningState | v1alpha1api20210101 |
-| PrivateEndpointConnectionProperties_Status                 | v1alpha1api20210101 |
-| PrivateEndpointConnection_Status                           | v1alpha1api20210101 |
-| PrivateEndpoint_Status                                     | v1alpha1api20210101 |
-| PrivateLinkServiceConnectionState_Status                   | v1alpha1api20210101 |
-| PrivateLinkServiceConnectionStatus_Status                  | v1alpha1api20210101 |
-| PublicNetworkAccessType_Status                             | v1alpha1api20210101 |
-| VirtualMachineFamilyCoreQuota_Status                       | v1alpha1api20210101 |
+| Types defined in package "batch"                  | v1alpha1api20210101 |
+|---------------------------------------------------|---------------------|
+| APIVersion                                        | v1alpha1api20210101 |
+| AutoStorageBaseProperties                         | v1alpha1api20210101 |
+| AutoStorageBaseProperties_Status                  | v1alpha1api20210101 |
+| BatchAccount                                      | v1alpha1api20210101 |
+| BatchAccountIdentity                              | v1alpha1api20210101 |
+| BatchAccountIdentityType                          | v1alpha1api20210101 |
+| BatchAccountIdentity_Status                       | v1alpha1api20210101 |
+| BatchAccountIdentity_StatusUserAssignedIdentities | v1alpha1api20210101 |
+| BatchAccountProperties                            | v1alpha1api20210101 |
+| BatchAccountProperties_Status                     | v1alpha1api20210101 |
+| BatchAccount_Spec                                 | v1alpha1api20210101 |
+| BatchAccount_Status                               | v1alpha1api20210101 |
+| EncryptionProperties                              | v1alpha1api20210101 |
+| EncryptionPropertiesKeySource                     | v1alpha1api20210101 |
+| EncryptionProperties_Status                       | v1alpha1api20210101 |
+| KeyVaultProperties                                | v1alpha1api20210101 |
+| KeyVaultProperties_Status                         | v1alpha1api20210101 |
+| KeyVaultReference                                 | v1alpha1api20210101 |
+| KeyVaultReference_Status                          | v1alpha1api20210101 |
+| PoolAllocationMode                                | v1alpha1api20210101 |
+| PublicNetworkAccessType                           | v1alpha1api20210101 |
```

Comment:
How come the list of generated types is so much shorter? 33 vs 21 is a large decrease.